src/AlgebraicGeometry/Schemes/Covering/Objects/Attributes.jl

```diff
@@ -46,3 +46,42 @@ function glueing_graph(C::Covering)
   return C.glueing_graph
 end
 
+@doc raw"""
+    decomposition_info(C::Covering)
+
+Return an `IdDict` `D` with the `patches` ``Uᵢ`` of `C` as keys and values a list 
+of elements ``fᵢ₁,…,fᵢᵣ ∈ 𝒪(Uᵢ)``. These elements are chosen so that for every 
+affine patch `Uᵢ` of ``X`` in the covering `C` the closed subvarieties ``Zᵢ ⊂ Uᵢ`` 
+defined by the ``fᵢⱼ`` give rise to a decomposition of ``X`` as a **disjoint** union 
+``X = \bigcup_{i} Z_i`` of locally closed subvarieties. 
```

Comment:
So $f_{i1}=0$?

Reply:
I still don't understand this.

Reply:
What is $Z_i$ exactly? I guess an example would help.

Reply:
$Z_i \subset U_i$ is the subvariety defined by the elements $f_{i,1}, \dots, f_{i,r}$ returned by the call `decomposition_info(X)[U_i]`.

Reply:
O.k. please add this to the docstring with the next pr.

Reply:
But still how do you get 

$X = \bigcup_{i} Z_i$?

Reply:
This must hold by construction of the `decomposition_info`. You do not get a decomposition with inclusion maps, etc. You get a list of ring elements for every patch and you may continue working with them while assuming that the decomposition described above is a disjoint union.

Reply:
Still don't get it. Because I do not get your definition of a decomposition_info. 
Please give clean mathematical definition.